gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-environments.yaml

```diff
@@ -716,6 +717,35 @@ components:
                 - UNDEPLOYED
                 - PENDING
 
+        CreateClusterDto:
+            type: object
+            properties:
+                name:
+                    type: string
+                description:
+                    type: string
+                configuration:
+                    type: object
+            required: [ "name", "configuration" ]
+
+        ClusterDto:
+            type: object
+            properties:
+                id:
+                    type: string
+                createdAt:
+                    type: string
+                    format: date-time
+                updatedAt:
+                    type: string
+                    format: date-time
+                name:
+                    type: string
+                description:
+                    type: string
+                configuration:
+                    type: object
+
```

Comment:
As it is a user documentation, I think it's better to keep `Cluster` and `CreateCluster` instead of suffixing with `Dto`.
Even if I understand the name overlapping with our service layer